README.md

```diff
@@ -1,3 +1,17 @@
+<h1 align="center">Movie Browser<h1>
+
+# This is a demo of our application, where you can see currently popular movies, actors and information about them
+
+[Demo Movie Browser](https://mateuszpiaszczyk.github.io/MoviesBrowser/)
+
+## Used technologies, languages
+
+- React
+- JavaScript
+- Redux-Saga
+
+## Further readme it will show up soon
```

Comment:
I would write:
"This readme wil be updating along with our aplication."
It sounds more pro and it is correct ;)

Reply:
Changes adjustment